webview_panels/src/modules/defer/DeferToProduction.tsx

```diff
@@ -240,13 +244,17 @@ const DeferToProduction = (): JSX.Element => {
     projectIntegrations: [],
     dbt_core_integration_id: -1,
   });
+  const [showPanel, setShowPanel] = useState(false);
 
   const onMesssage = useCallback(
     (event: MessageEvent<IncomingMessageProps>) => {
       const { command, args } = event.data;
       switch (command) {
         case "updateDeferConfig":
-          setDeferState(args as unknown as DeferToProductionProps);
+          if (args) {
```

Comment:
here you are checking falsy value for args and if there is no project you are sending `""` from webview provider. so this if loop will not be executted right? then the deferstate variable will not be unset.